core/driver/sources/src/main/resources/reference.conf

```diff
@@ -282,12 +282,19 @@ probe {
       freezes.enabled = false
     }
 
-    // `driver.display` defines the display mode which should be used for the IDE run. By default, it is "native",
-    // which means that everything is displayed on the screen and you can interact with the IDE during test.
-    // Another possible value is "xvfb" which makes the IDE run in the xvfb mode (using virtual display; only available
-    // under X Window System). This config can be set either with a raw string value or by setting the IDEPROBE_DISPLAY
-    // environment variable which overrides the default value.
-    display = "native" # OR "xvfb" - these are two possible values
+    // `driver.display` defines the display mode which should be used for the IDE run.
+    // - By default, it is "native", which means that everything is displayed on the screen and you can interact
+    //   with the IDE during test.
+    // - Second possible value is "xvfb" which makes the IDE run in the xvfb mode (using virtual display; only available
+    //   under X Window System). This config can be set either with a raw string value or by setting the IDEPROBE_DISPLAY
```

Comment:
` This config can be set either with a raw string value or by setting the IDEPROBE_DISPLAY...` applies to the entire `display` config key, not just to the `Second possible value`

Reply:
Yeah, my mistake. Will be fixed 👍